Add security section (#191)
PrestaShop/open-source: content/security/security-policy.md

---
title: Security policy
---

# Security Policy

> Obviously, PrestaShop project security is a critical matter. PrestaShop teams are dedicated to keep a high level of security in every aspects of the software.

Reviewer: ```suggestion
Obviously, PrestaShop project security is a critical matter. Project members are dedicated to keeping a high level of security in every aspect of the software.
```

However a software without vulnerability does not exist, which is why there is a security report process. If you find a security issue, please follow it to [responsibly disclose](https://en.wikipedia.org/wiki/Responsible_disclosure) your findings.

## Reporting a Vulnerability

Security issues can be reported by sending an email to security-core@prestashop.com or through our [Bug Bounty Program][bug-bounty], which will go to security team members.

When the security team receives a security bug report, the report will be assigned to a primary handler. 
This person will coordinate the fix and release process, involving the following steps:

 - Confirm the problem and determine the affected versions.

>  - Audit code to find any potential similar problems.

Reviewer 2: ```suggestion
 - Audit the code to find any potential similar problems.
```

 - Prepare fixes for all releases still under maintenance. These fixes will be released as fast as possible.

The security team will follow up with a response indicating the next steps in handling the report.

> If the issue is confirmed, the security team will keep you informed of the progress towards a fix and full announcement, and may ask for additional information or guidance.

Reviewer 2: ```suggestion
If the issue is confirmed, the security team will keep you informed of the progress towards a fix, a full announcement, and may ask for additional information or guidance.
```
